pytorch_blade/pytorch_blade/compiler/mlir/runtime/ral_context.cpp

```diff
@@ -221,14 +221,16 @@ at::List<at::Tensor> RalContext::CreateAndBindingOutputs(
       out_tensor = torch::from_blob(
           const_cast<void*>(out_buf->data()),
           out_buf->shape(),
+          out_buf->strides(),
           deleter,
           option);
       out_buf->release();
     } else {
-      out_tensor =
-          torch::from_blob(
-              const_cast<void*>(out_buf->data()), out_buf->shape(), option)
-              .clone();
+      out_tensor = torch::from_blob(
```

Comment:
Don't you need to copy the underline buffer? @wyzero Pls double check if it is safe.